src/condense_loopset.jl

```diff
@@ -327,7 +337,8 @@ function setup_call_noinline(ls::LoopSet, U = zero(Int8), T = zero(Int8))
         push!(q.args, ex)
     end
     insert!(call.args, 8, Expr(:call, Expr(:curly, :Val, vptrarrays)))
-    insert!(call.args, 9, Expr(:call, Expr(:curly, :Val, vptrsubsetdims)))
+    # insert!(call.args, 9, Expr(:call, Expr(:curly, :Val, vptrsubsetdims)))
+    insert!(call.args, 9, vptrsubsetdims)
```

Comment:
This function parsed `Val{N}()` expressions to extract the `N`, and create a expression for `Val{tuple_of_Ns}()`. This broke as soon as the `Val{N}()` expressions were replaced with calls that returned `Val`s.
So now, instead of parsing expressions, it makes them arguments to a function that will return the desired answer.

The purpose of that song and dance is so that `__avx__!` is not inlined, and calls `_avx_!`, which is inlined.
Within the `__avx__!` call, it'll subset the arrays, get pointers, etc, so that LLVM still has this aliasing / relative pointer offset information available. This reduced the performance disparity between setting `inline=false` and `inline=true`. The latter is the default.
This is not a good solution, but a hack added on later when I decided to add this.